README.md

```diff
@@ -182,9 +178,18 @@ Removing nebula-docker-compose_metad2_1    ... done
 Removing network nebula-docker-compose_nebula-net
 ```
 
-**Note**: As your data is stored in your local computer, your data will be reserved even after you stop the services of **Nebula Graph**.
+## Other Ways to Install Nebula Graph
+
+* [Using Source Code](https://docs.nebula-graph.io/manual-EN/3.build-develop-and-administration/1.build/1.build-source-code/)
+* [Using Docker](https://docs.nebula-graph.io/manual-EN/3.build-develop-and-administration/1.build/2.build-by-docker/)
+* [Using .rpm or .deb Files](https://docs.nebula-graph.io/manual-EN/3.build-develop-and-administration/2.install/1.install-with-rpm-deb/)
+
+## FAQ
+
+**Q**: How to update the nebula-console client?
+
+**A**: Use the `docker-compose pull` command in the `nebula-docker-compose` directory on your host.
```

Comment:
This command is to update nebula service images, not console image. The right command is like:

```
docker pull vesoft/nebula-console:nightly   # v1.0
docker pull vesoft/nebula-console:v2-preview-nightly  # v2-preview
```

Reply:
Thank you for the heads-up!